maintainers/maintainer-list.nix

```diff
@@ -2824,6 +2824,16 @@
     githubId = 743057;
     name = "Danylo Hlynskyi";
   };
+  danc86 = {
+    name = "Dan Callaghan";
+    email = "djc@djc.id.au";
+    github = "danc86";
+    githubId = 398575;
+    keys = [{
+      longkeyid = "rsa4096/0x26B5AA2FDAF2F30A";
```

Comment:
```suggestion
```

this seems to be causing ofborg eval errors

Reply:
Ah, right, seems `longkeyid` has been removed on `master` since I originally wrote this commit. I've amended it to drop `longkeyid`.